Documentation/markdown/SetupHelp.md

```diff
@@ -1,126 +1,147 @@
+* Build all projects within the *Copy* project folder. 
 
-	**NOTE:** This must be done only for the first time. 
+*NOTE:* This only needs to happen when compiling for the first time, or after a deletion of the binaries folder.
 
-2. After first project is successfully built find the folder **ExpressBindingGenerator**. *Build* project **OpenInfraPlatform.ExpressBindingGenerator**.
+### <a name="generating_EarlyBinding"></a> Generating IFC early binding libraries
 
-	**HINT:** You can expect that one item will be skipped. This item will be *built* afterwards. 
+*NOTE:* If you are using OpenInfraPlatform only with point clouds, you can skip these steps.
 
-3. In the folder **ExpressBindingGenerator** find the folder **Commands**. There you should *build* project **Commands.GenerateEarlyBinding.IFC?**. 
+1. Find the folder *ExpressBindingGenerator*. Build project **OpenInfraPlatform.ExpressBindingGenerator**.
 
-	**NOTE:** By the default this will build **Commands.GenerateEarlyBinding.IFC4X3**. If you change a shema to other, you would have to *generate* CMake again.
+2. In the folder *ExpressBindingGenerator* find the folder *Commands*. 
+   There you should build project **Commands.GenerateEarlyBinding.IFC?** where *?* stands for the chosen IFC version.
 
-4. **Important**: Open CMake and select *Generate* to include newly generated IFC early binding code in the solution.
+	*NOTE:* By default settings, this will build **Commands.GenerateEarlyBinding.IFC4X3_RC1**. 
+    If you want to change the schema to another, you need to select the corresponding option in CMake and *generate* in CMake again.
+
+3. **Important**: Now open CMake and select *Generate* to include newly generated IFC early binding code in the solution.
+
+4. Find the folder *EarlyBinding*. Build project **OpenInfraPlatform.IFC?**.
 
 ### <a name="Compiling_interface"></a> Compiling user interface
 
-**NOTE:** Build dependencies are set, so building only the last step should automatically build all.
+*NOTE:* Build dependencies are set through CMake, so building only the last step should automatically build all.
 
-1. Open Visual Studio and find the folder **EarlyBinding**. *Build* project **OpenInfraPlatform.IFC?**.
 1. If you are using point clouds *build* project **OpenInfraPlatform.PointCloudProcessing**.
 1. *Build* project **OpenInfraPlatform.Core**.
 1. *Build* project **OpenInfraPlatform.Rendering**.
 1. *Build* project **OpenInfraPlatform.UI**.
-1. *Build* all projects staged in the **Copy** project folder. 
-
-	**NOTE:** execute **Get_OKLABI.cmd** in the source folder (e.g. `C:\dev\Open-Infra-Platform\external`) to prevent an error during the build process. After the execution of **Get_OKLABI.cmd**, it's necessary to *configure* and *generate* OpenInfraPlatform project once again in CMake.
 
 
 ## <a name="Generating_Doc"></a> Generating documentation
 
 ### <a name="Get_Dot"></a> Get Dot and Doxygen
```

Comment:
Maybe it will be better to place **Get Dot and Doxygen** before **Generating solution**, since CMake can't configure OIP without Executing GetDot.cmd and GetDoxygen.cmd